packit.spec

```diff
@@ -2,7 +2,7 @@
 %global real_name packit
 
 Name:           %{real_name}
-Version:        0.12.0
+Version:        0.13.0
```

Comment:
You're missing a `%changelog` entry as well.
Try `rpmdev-bumpspec packit.spec`. (from `rpmdevtools` rpm)